src/AppBundle/Service/Notification/Channel/MailingChannel.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace AppBundle\Service\Notification\Channel;
+
+use AppBundle\Model\User\User;
+use AppBundle\Service\Notification\NotificationInput;
+use Swift_Mailer;
+use Symfony\Component\Templating\EngineInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * MailingChannel.
+ *
+ * @author Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * @author Ben Bieler <benjaminbieler2014@gmail.com>
```

Comment:
@benbieler I keep you as author here as that's actually the old `MailListener` you built in #130, but GIT doesn't recognize this as moved file...